src/utils/index.js

```diff
@@ -1,8 +1,10 @@
-
 function isFinite(value) {
-	return !(typeof value !== 'number' ||
-	value !== value || value === Infinity ||
-	value === -Infinity);
+	return !(
```

Comment:
why not
```js
	return (
		typeof value === 'number' &&
		value === value &&
		value !== Infinity &&
		value !== -Infinity
	);
```
?

Reply:
Yeah, your version is more readable, will swap them